af_pipeline/archive/deprecated_funcs.py

```
@@ -0,0 +1,61 @@
+
+def get_interacting_patches(
+    self,
+    contact_map: np.array,
```

Ruff (0.8.2)

Undefined name `np`

(F821)